src/vaadin-message.js

```diff
@@ -74,19 +74,23 @@ class MessageElement extends ElementMixin(ThemableMixin(PolymerElement)) {
           display: none !important;
         }
 
-        .vaadin-message-wrapper {
+        [part='content'] {
           display: flex;
           flex-direction: column;
-          flex: 1;
+          flex-grow: 1;
```

Comment:
Add 
```
vaadin-avatar {
  flex-shrink: 0
}
```
To avoid that the avatar gets squeezed when the message needs multiple rows and thus part="content" starts expanding upon avatar due to this row.

Reply:
So peculiar, I tested this with Chrome, Safari, FF and Edge and it never "shrunk". :/ Added though.

Reply:
Did you test it with a narrow window enough? It happened only when wordwrap happened. Anyhow, good that it got fixed.